src/alchemtest/namd/tyr2ala/descr.rst

```diff
@@ -0,0 +1,4 @@
+=======================================================================
+ NAMD: free energy of tyrosine to alanine mutation in aqueous solution
+=======================================================================
+
```

Comment:
@vtlim please add details to the description. See the existing `descr.rst` files such as  [amber/simplesolvated/descr.rst](https://github.com/alchemistry/alchemtest/blob/master/src/alchemtest/amber/simplesolvated/descr.rst).

Most importantly: it needs to have a *license* and the  *donor* should be who can grant the license.

Reply:
According to the discussion https://github.com/alchemistry/alchemtest/issues/17#issuecomment-349471034 , the donor should be JC Gumbart and the license is CC0, or has anything changed?

If the source is an existing file (NAMD tutorial??) then please also add to the text where you got the files from.

Reply:
@orbeckst -- done. The donor is JC Gumbart, yes. Is there place where the input files should go?

Reply:
Thanks for adding the meta data.

> Is there place where the input files should go?

We don't store input files here. If there's a web page where people could go to or if you want to put the files on zenodo or figshare or GitHub then that URL could be added to the text. 

Is there a URL you want to add right now?